src/pages/topic-stuff/flashcard/flashcard.css

```diff
@@ -146,7 +146,7 @@
 }
 
 .flashcard-title {
-    font-family: var(--current-font);
+    font-family: var(--font-default);
```

Comment:
I think the font should be bolder, to highlight the the important stuff, like headings more

Reply:
Yes.